src/header/main.tsx

```diff
@@ -34,10 +34,11 @@ const getQueryParam = (fullUrl = "", variableToFind: string) => {
     } else {
       document.addEventListener("DOMContentLoaded", fn);
     }
-    // This is to set the font size to 1em on the HTML element for sites
-    // that may have a different base font size, such as "62.5%".
-    if (window.location.origin.includes("vega")) {
-      document.getElementsByTagName("html")[0].style.fontSize = "1em";
+    // Sets the base font size to 16px for libguides.nypl.org;
+    // (it's currently set to 10px which interferes with how the header
+    // is meant to look)
+    if (window.location.origin.includes("libguides")) {
+      document.getElementsByTagName("html")[0].style.fontSize = "16px";
```

Comment:
Is it not possible to make this change on the LibGuides end?

Reply:
I think it's possible, and Edwin and I discussed it as a second choice if this option didn't work. I think we tried this first because it was within our control.. Do you know who we would contact if we wanted to make this change there?

Reply:
Going to merge for now to move this along, but happy to reach out to libguides in future if we think that's a better path forward.